src/main/java/training/subsystems/elbow/ElbowConstants.java

```diff
@@ -0,0 +1,47 @@
+package training.subsystems.elbow;
+
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ElbowConstants {
+
+	public enum PresetPositions {
+
+		SAFE(Rotation2d.fromDegrees(0)),
+		INTAKE(Rotation2d.fromDegrees(0)),
+		SCORE(Rotation2d.fromDegrees(0)),
+		STARTING(Rotation2d.fromDegrees(0)),
+		TRANSFER(Rotation2d.fromDegrees(0));
+
+		public final Rotation2d ANGLE;
+
+		PresetPositions(Rotation2d angle) {
+			this.ANGLE = angle;
+		}
+
+	}
+	public static final int MOTOR_ID = 5;
+	public static final int P = 10;
+
+	public static final int I = 1;
```

Comment:
set this to 0, an I of 1 will make the elbow move really fast after a few seconds